pkg/auth/auth.go

```diff
@@ -134,9 +134,13 @@ func Login(ctx context.Context, systemContext *types.SystemContext, opts *LoginO
 
 	if err = docker.CheckAuth(ctx, systemContext, username, password, server); err == nil {
 		// Write the new credentials to the authfile
-		if err := config.SetAuthentication(systemContext, server, username, password); err != nil {
+		path, err := config.SetCredentials(systemContext, server, username, password)
+		if err != nil {
 			return err
 		}
+		if opts.Verbose {
+			fmt.Fprintln(opts.Stdout, "Created: ", path)
```

Comment:
I am not a fan of Created:, since this may not have created anything.  Lets just change the output do 
fmt.Println(opts.Stdout, desc)

Reply:
“Stored in: $location”? “Used: $location”? “Updated: $location”? A lone path without any context might feel like a debugging oversight.

Reply:
I agree with @mtrmac , we should use `Used: %s` or `Updated: %s` otherwise it would be context-less. WDYT ?

Reply:
@rhatdan Added change on latest commit.

Reply:
I am fine with "Stored in" or Used, better then Created.